.gitignore

```diff
@@ -39,3 +39,10 @@ lib
 test
 tools/typings/tsd
 tmp
+
+# Transpiled files
+gulpfile.js*
+app/**/*.js
```

Comment:
Have deliberately not done '*.js*' - this also excludes '.json' files. It may or may not be an issue.